<?xml version="1.0"?>
<launch>
   <arg name="model" default="$(find hector_quadrotor_description)/urdf/quadrotor_with_cam.gazebo.xacro" />


   <include file="$(find hector_gazebo_worlds)/launch/rolling_landscape_120m.launch"/>

   <group ns="uav1">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav1" />
       <arg name="tf_prefix" value="uav1" />
       <arg name="model" value="$(arg model)" />
       <arg name="y" value="-2.0" />
     </include>
   </group>

   <group ns="uav2">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav2" />
       <arg name="tf_prefix" value="uav2" />
       <arg name="model" value="$(arg model)" />
       <arg name="y" value="2.0" />
     </include>
   </group>

   <group ns="uav0">
      <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch" >
        <arg name="name" value="uav0" />
        <arg name="tf_prefix" value="uav0" />
        <arg name="model" value="$(arg model)"/> 
      </include>
   </group>
  
   <node pkg="rviz" type="rviz" name="rviz" args="-d $(find visualuwb)/rviz/multiquadcopter.rviz">
    </node>
  

</launch>
